hook.php

```diff
@@ -288,6 +288,11 @@ function plugin_tag_getRuleActions($params = [])
                 'name'  => __("Add tags", 'tag'),
                 'type'  => 'dropdown',
                 'table' => PluginTagTag::getTable(),
+                'force_actions' => [
+                    0 => "assign",
+                    1 => "append"
+                ],
```

Comment:
It's not necessary to index the array:
```suggestion
                'force_actions' => ['assign', 'append'],
```